web/base_viewer.js

```diff
@@ -936,8 +969,42 @@ class BaseViewer {
       };
     });
   }
+
+  setScrollMode(mode) {
+    this._scrollMode = mode;
+    this.viewer.classList.remove('scrollHorizontal', 'scrollWrapped');
+    this._isHorizontallyScrolling = false;
+    switch (mode) {
+      case ScrollMode.HORIZONTAL:
+        this._isHorizontallyScrolling = true;
+        this.viewer.classList.add('scrollHorizontal');
+        break;
+      case ScrollMode.WRAPPED:
+        this.viewer.classList.add('scrollWrapped');
+        break;
+    }
```

Comment:
Similarly as my other comment about `classList`, the code can be simplified to:
```javascript
this._isHorizontallyScrolling = mode === ScrollMode.HORIZONTAL;
this.viewer.classList.toggle('scrollHorizontal', mode === ScrollMode.HORIZONTAL);
this.viewer.classList.toggle('scrollWrapped', mode === ScrollMode.WRAPPED);
```

Reply:
Will do.